plugins/inputs/win_services/win_services.go

```diff
@@ -0,0 +1,138 @@
+// +build windows
+
+package win_services
+
+import (
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"golang.org/x/sys/windows/svc/mgr"
+    "fmt"
+)
+
+var sampleConfig = `
+  ## Names of the services to monitor. Leave empty to monitor all the available services on the host
+  service_names = [
+    "LanmanServer",
+    "TermService",
+  ]
+`
+
+var description = "Input plugin to report Windows services info: service name, display name, state, startup mode"
+
+type Win_Services struct {
+	ServiceNames []string `toml:"service_names"`
+}
+
+type ServiceInfo struct {
+	ServiceName		string
+	DisplayName		string
+	State      		int
+	StartUpMode     int
+    Error           error
+}
+
+var  ServiceStatesMap = map [int]string{
+    0x00000001: "stopped",
+    0x00000002: "start_pending",
+    0x00000003: "stop_pending",
+    0x00000004: "running",
+    0x00000005: "continue_pending",
+    0x00000006: "pause_pending",
+    0x00000007: "paused",
+}
+
+var  ServiceStartupModeMap = map [int]string{
+    0x00000000: "boot_start",
+    0x00000001: "system_start",
+    0x00000002: "auto_start",
+    0x00000003: "demand_start",
+    0x00000004: "disabled",
+
+}
+
+func (m *Win_Services) Description() string {
+	return description
+}
+
+func (m *Win_Services) SampleConfig() string {
+	return sampleConfig
+}
+
+
+func (m *Win_Services) Gather(acc telegraf.Accumulator) error {
+
+    serviceInfos, err := listServices(m.ServiceNames)
+
+    if err != nil {
+        return err
+    }
+
+    for _, service := range serviceInfos {
+        if service.Error == nil {
+            fields := make(map[string]interface{})
+            tags := make(map[string]string)
+
+            tags["display_name"] = service.DisplayName
+            tags["service_name"] = service.ServiceName
+
+            fields["state"] = ServiceStatesMap[service.State]
+            fields["startup_mode"] = ServiceStartupModeMap[service.StartUpMode]
```

Comment:
Need to do something smart if the value returned is not in the map, probably return an error and skip the service.  This could be tested earlier on in listServices.